.github/workflows/e2e.yaml

```diff
@@ -13,8 +13,7 @@ jobs:
     strategy:
       matrix:
        kube-version:
-       - "1.21"
-       - "1.22"
+       - "1.19"
```

Comment:
So, no explicit test on 1.21 and 1.22? It might be good to document the reasoning.

Reply:
added a comment to the workflow